pytest-embedded-idf/pytest_embedded_idf/unity_tester.py

```diff
@@ -422,23 +422,47 @@ def run_all_single_board_cases(
         reset: bool = False,
         timeout: float = 30,
         run_ignore_cases: bool = False,
+        names: t.Optional[list] = None,
+        attributes: t.Optional[dict] = None,
     ) -> None:
         """
         Run all single board cases, including multi_stage cases, and normal cases
 
         Args:
-            group: test case group
-            reset: whether to perform a hardware reset before running a case
-            timeout: timeout. (Default: 30 seconds)
-            run_ignore_cases: run ignored test cases or not
+            group: Test case group. Supports group invert by '!' and groups separated by ','.
+            reset: Whether to perform a hardware reset before running a test case.
+            timeout: Timeout duration in seconds. Default is 30 seconds.
+            run_ignore_cases: Whether to run ignored test cases or not.
+            names: List of test case names to run.
```

Comment:
```suggestion
            name: test case name or a list of test case names to run.
```

same as `group`, better support both str and list[str]